src/io/flutter/module/settings/SettingsHelpForm.java

```diff
@@ -50,27 +47,15 @@ public SettingsHelpForm() {
     orgLabel.setText(FlutterBundle.message("flutter.module.create.settings.help.org.label"));
     orgDescription.setText(FlutterBundle.message("flutter.module.create.settings.help.org.description"));
 
-    gettingStartedUrl.setText(FlutterBundle.message("flutter.module.create.settings.help.getting_started_html"));
+    gettingStartedUrl.setText(FlutterBundle.message("flutter.module.create.settings.help.getting_started_link_text"));
     gettingStartedUrl.setCursor(Cursor.getPredefinedCursor(Cursor.HAND_CURSOR));
-    gettingStartedUrl.addMouseListener(new MouseAdapter() {
-      public void mouseClicked(MouseEvent e) {
-        if (e.getClickCount() > 0) {
-          if (Desktop.isDesktopSupported()) {
-            Desktop desktop = Desktop.getDesktop();
-            try {
-              URI uri = new URI("https://flutter.io/getting-started/");
-              desktop.browse(uri);
-            } catch (IOException | URISyntaxException ex) {
-              // do nothing
-            }
-          } else {
-            //do nothing
-          }
-        }
-      }
-    });
+    gettingStartedUrl.setIcon(null);
+    //noinspection unchecked
+    gettingStartedUrl
```

Comment:
👍 

When things play nice with IntelliJ APIs, it's amazing how terse things can get!